### PR TITLE
Login Page Ready

### PR DESCRIPTION
**Pull request by ronakp**
Thanks to https://www.youtube.com/watch?v=iYH2jzUM1Nc
Nice Video

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
...
+class LoginPageState extends State<LoginPage>{
 
-  // This class is the configuration for the state. It holds the values (in this
-  // case the title) provided by the parent (in this case the App widget) and
-  // used by the build method of the State. Fields in a Widget subclass are
-  // always marked "final".
+  //Animation is not need as of now.
 
-  final String title;
+  /*AnimationController animationController;
```
Comment (nirav076) [inline lib/main.dart]: From line 26 to 42 we have commented code. As it is not needed we should remove it. We can anyway go back to previous commits if needed for reference to this code.

File: lib/main.dart [hunk 2]
```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
...
-      ),
-      floatingActionButton: new FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: new Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
     );
   }
-}
+}
```
Comment (nirav076) [inline lib/main.dart]: Convert it to windows file format as it was. Look for how to do it for specific editor/platform you are using and update it here also for others.
Refer this: https://kb.iu.edu/d/acux

File: lib/main.dart [hunk 3]
```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
 void main() => runApp(new MyApp());
 
 class MyApp extends StatelessWidget {
```
Comment (nirav076) [inline lib/main.dart]: Should we decide a name and update it in first commit itself?

File: lib/main.dart [hunk 4]
```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
...
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return new MaterialApp(
-      title: 'Flutter Demo',
```
Comment (nirav076) [inline lib/main.dart]: We have removed title. Should it be kept with appropriate value for completeness?

**Review by nirav076 — COMMENTED** [review]
Have added some review comments based on best practices I have learned at App**bit :innocent:. 
Pardon me, if it feels over-killing :sweat_smile: then reply to comments accordingly if we should not do it at the moment.

**Comment by nirav076** [comment]
Haven't tested it yet. Will re-review after setting up the environment run it.

File: lib/main.dart (hunk 1 above)
Comment (ronakp) [reply lib/main.dart]: yes good idea. I have removed it.

File: lib/main.dart (hunk 3 above)
Comment (ronakp) [reply lib/main.dart]: rename it to ProjectNeo. sounds good?

File: lib/main.dart (hunk 2 above)
Comment (ronakp) [reply lib/main.dart]: What is this for? I am using android studio and I can export my formatter.

File: lib/main.dart (hunk 4 above)
Comment (ronakp) [reply lib/main.dart]: It doesn't matter. our app does not have the title anyway.

File: lib/main.dart (hunk 2 above)
Comment (nirav076) [reply lib/main.dart]: The repo must have been originally coded on unix system where new line does not contain <enter> ASCII char. And now when you saved it through windows system, it added it. Thus though it is not visible, there is change in this line.
Anyway, that's not big issue if we will be continuing on windows. No need to change. 
BTW where did we got this source repo from? Or you created a project and cheked it in?

File: lib/main.dart (hunk 3 above)
Comment (nirav076) [reply lib/main.dart]: Ya, that fine.

File: lib/main.dart (hunk 4 above)
Comment (nirav076) [reply lib/main.dart]: Fine.

**Review by nirav076 — APPROVED** [review]
Have one doubt regarding from where we got this source repo. Just reply on that. Otherwise LGTM.

**Pull request merged**